### ISSUE TITLE
(3) Invalid aria attribute in Blacklight modal

### ISSUE THREAD

**Issue by rschwab** [UCSCLibrary/dams_project_mgmt#260]
There is an aria-labelledby attribute with an invalid value in Blacklight. Its an accessibility issue we should correct.

The file is bundle/gems/blacklight-6.21.0/app/views/shared/_ajax_modal.html.erb

The snippet is:
`<div id="ajax-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="modal menu" aria-hidden="true">`

The aria-labelledby should refer to an entity ID, but I'm not familiar enough with what Blacklight is doing to offer a solution, yet.

**Comment by rschwab** [comment]
Resolved in the Blacklight project with this commit: https://github.com/projectblacklight/blacklight/commit/c569c2cf8743bfc2ac1453685b8fae7954034415

**Comment by chesleyperry** [comment]
I ran the some pages from the staging site through the SiteImprove Chrome plugin and didn't see this error, so I'm assuming it's fixed.
